### ISSUE TITLE
[Help Wanted] [PLT-6710] /rename [text] slash command: Rename the channel

### ISSUE THREAD

**Issue by esethna** [mattermost/platform#6558]
If you're interested please comment here and come [join our "Contributors" community channel](https://pre-release.mattermost.com/core/channels/tickets) on our daily build server, where you can discuss questions with community members and the Mattermost core team. For technical advice or questions, please  [join our "Developers" community channel](https://pre-release.mattermost.com/core/channels/developers).

New contributors please see our [Developer's Guide](https://docs.mattermost.com/guides/developer.html), specifically for [machine setup](https://docs.mattermost.com/developer/developer-setup.html) and for [developer workflow](https://docs.mattermost.com/developer/developer-flow.html).

----

**Notes**: [Jira ticket](https://mattermost.atlassian.net/browse/PLT-6710)

/rename [text] slash command: Rename the channel

Notes:
1. /rename [text] will set the channel display name to "[text]"
2. Permission to use this command is set by the [public/private channel management policy settings|https://docs.mattermost.com/administration/config-settings.html#policy]. If the user enters "/header [text]" without permissions, post this ephemeral message:

```
You do not have the appropriate permissions to rename the channel.
```

3. Autocomplete: 
```
/rename [text] 
Rename the channel.
```

**Comment by cpanato** [comment]
I will work on that

**Comment by cpanato** [comment]
PR: https://github.com/mattermost/platform/pull/6570

**Comment by cpanato** [comment]
this can be closed due #6570 is already merged
